contracts/dao/voting/investors-vesting-vault/src/contract.rs

```diff
@@ -0,0 +1,216 @@
+    };
+    let manager = msg
+        .manager
+        .map(|manager| deps.api.addr_validate(&manager))
+        .transpose()?;
+
+    let vesting_contract_address = deps.api.addr_validate(&msg.vesting_contract_address)?;
+
+    let config = Config {
+        vesting_contract_address,
+        description: msg.description,
+        owner,
+        manager,
+    };
+
+    CONFIG.save(deps.storage, &config)?;
+    DAO.save(deps.storage, &info.sender)?;
+
+    Ok(Response::new()
+        .add_attribute("action", "instantiate")
+        .add_attribute("description", config.description)
+        .add_attribute("vesting_contract_address", config.vesting_contract_address)
+        .add_attribute("owner", config.owner)
+        .add_attribute(
+            "manager",
+            config
+                .manager
+                .map(|a| a.to_string())
+                .unwrap_or_else(|| "None".to_string()),
+        ))
+}
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn execute(
+    deps: DepsMut,
+    _env: Env,
+    info: MessageInfo,
+    msg: ExecuteMsg,
+) -> Result<Response, ContractError> {
+    match msg {
+        ExecuteMsg::UpdateConfig {
+            vesting_contract_address,
+            owner,
+            manager,
+            description,
+        } => execute_update_config(
+            deps,
+            info,
+            vesting_contract_address,
+            owner,
+            manager,
+            description,
+        ),
+    }
+}
+
+pub fn execute_update_config(
+    deps: DepsMut,
+    info: MessageInfo,
+    new_vesting_contract_address: Option<String>,
+    new_owner: String,
+    new_manager: Option<String>,
+    new_description: Option<String>,
+) -> Result<Response, ContractError> {
+    let mut config: Config = CONFIG.load(deps.storage)?;
+    if info.sender != config.owner && Some(info.sender.clone()) != config.manager {
+        return Err(ContractError::Unauthorized {});
+    }
+
+    let new_vesting_contract_address = new_vesting_contract_address
+        .map(|new_vesting_contract_address| deps.api.addr_validate(&new_vesting_contract_address))
+        .transpose()?;
+
+    let new_owner = deps.api.addr_validate(&new_owner)?;
+    let new_manager = new_manager
+        .map(|new_manager| deps.api.addr_validate(&new_manager))
+        .transpose()?;
+
+    if info.sender != config.owner && new_owner != config.owner {
+        return Err(ContractError::OnlyOwnerCanChangeOwner {});
+    };
+
+    config.owner = new_owner;
+    config.manager = new_manager;
+    if let Some(description) = new_description {
+        config.description = description;
+    }
+    if let Some(new_vesting_contract_address) = new_vesting_contract_address {
+        config.vesting_contract_address = new_vesting_contract_address;
+    }
+
+    CONFIG.save(deps.storage, &config)?;
+    Ok(Response::new()
+        .add_attribute("action", "update_config")
+        .add_attribute("description", config.description)
```

Comment:
in order to prevent empty attribute errors please add and use config validation as here: https://github.com/neutron-org/neutron-dao/blob/feat/NTRN-380_vesting_vault/contracts/dao/voting/vesting-lp-vault/src/contract.rs#L171

in this handler and in the instantiate